### ISSUE TITLE
Stop automatic sorting during and after onchange

### ISSUE THREAD

**Issue by rerraw** [select2/select2#4766]
I used the select2 plugin within my WordPress theme, in Menu items page. I used this code below until I sort items or tags as I select them. The code work with me only during selection means during (on change trigger) but when I click on Save settings button in my WordPress menu item to save changes after reloading page I see the tags sorted automatically and not sorted like as I selected them during on change

    $(".bsnselect").select2();
    $(".bsnselect").on("select2:select.bsnselect", function (evt) {
         var element = evt.params.data.element;
         var $element = $(element);
         $element.detach();
         $(this).append($element);
         $(this).trigger("change");
    });

**Comment by rerraw** [comment]
I used the select2 plugin within my WordPress theme, in Menu items page. I used this code below until I sort items or tags as I select them. The code work with me only during selection means during (on change trigger) but when I click on Save settings button in my WordPress menu item to save changes after reloading page I see the tags sorted automatically and not sorted like as I selected them during on change
    
     $(".bsnselect").select2();
     $(".bsnselect").on("select2:select.bsnselect", function (evt) {
             var element = evt.params.data.element;
             var $element = $(element);
             $element.detach();
             $(this).append($element);
             $(this).trigger("change");
    });

**Comment by kevin-brown** [comment]
This is a duplicate of https://github.com/select2/select2/issues/3106.
